src/bosh-stemcell/spec/os_image/ubuntu_trusty_spec.rb

```diff
@@ -303,8 +303,14 @@
   end
 
   context 'PAM configuration' do
-    describe file('/lib/x86_64-linux-gnu/security/pam_cracklib.so') do
-      it { should be_file }
+    if Bosh::Stemcell::Arch.ppc64le?
```

Comment:
yep, just waiting for https://github.com/cloudfoundry/bosh-davcli/pull/6, to do all updates in one commit